apps/css-workshop/src/pages/menu.astro

```diff
@@ -1082,58 +1275,66 @@ import Icon_ from '../components/Icon.astro';
-      <li class='iui-list-item iui-large' data-iui-actionable='true' role='menuitem' tabindex='0'>
-        <svg-placeholder class='iui-list-item-icon' aria-hidden='true'></svg-placeholder>
-        <span class='iui-list-item-content'>
-          <div>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-            incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
-            exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
-            dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-            mollit anim id est laborum.
-          </div>
-          <div class='iui-list-item-description'>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-            incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
-            exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
-            dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-            mollit anim id est laborum.
-          </div>
-        </span>
-        <svg-caret-right-small class='iui-list-item-icon' aria-hidden='true'
-        ></svg-caret-right-small>
-      </li>
+        <svg-placeholder slot='start-icon'></svg-placeholder>
+         Alpha
+        <div class='iui-list-item-description'>Lorem ipsum</div>
+        <svg-checkmark slot='end-icon'></svg-checkmark>
+      </ListItem_>
+      <ListItem_
+        contentProps={{ class: 'iui-list-item-content' }}
+        size='large'
+        actionable
+        role='menuitem'
+        tabindex='0'
+      >
+        <svg-placeholder slot='start-icon'></svg-placeholder>
+         Beta
+        <div class='iui-list-item-description'>Lorem ipsum</div>
+        <svg-caret-right-small slot='end-icon'></svg-caret-right-small>
+      </ListItem_>
+      <ListItem_
+        contentProps={{ class: 'iui-list-item-content' }}
+        size='large'
+        actionable
+        role='menuitem'
+        tabindex='0'
+      >
+        <svg-placeholder slot='start-icon'></svg-placeholder>
+        <div>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+          ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+          ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+          sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+          est laborum.
+        </div>
+        <div class='iui-list-item-description'>
```

Comment:
Looks like description is not being handled by `ListItem` right now

Reply:
I have put description and list content base class inside `ListItem`.